app/users/user.model.ts

```diff
@@ -63,7 +63,7 @@ export class User {
    * @returns {User}
    */
   static getEmptyUser() {
-    return new User('', '', '', '', '', '');
+    return new User('', '', '', '', [''], '');
```

Comment:
Maybe just use `[]` here instead of `['']`

Reply:
okay..i changed this one..